server/netty-server/src/main/scala/sttp/tapir/server/netty/internal/NettyServerHandler.scala

```diff
@@ -212,6 +213,14 @@ class NettyServerHandler[F[_]](
         ctx.writeAndFlush(res, channelPromise).closeIfNeeded(req)
 
       },
+      wsHandler = (channelPromise) => {
```

Comment:
shouldn't this be a 400 as well? would correspond to the case where a ws endpoint is called without the ws upgrade headers, no?

Reply:
actually maybe we should have tests, that both calling a WS endpoint normally, and calling with WS a normal endpoint end up with a 400